hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/state/virtual/ContractKeyTest.java

```diff
@@ -55,6 +55,10 @@ class ContractKeyTest {
 
     // The first two columns form a ContractKey(id, 32-byte word) and the third column is the
     // result of running ContractKey.hashCode() on that key using the v0.35.3 tag.
+    //
+    // Important! These values should never be changed! If this test fails in
```

Comment:
Might be a good place for wingdings like  **🛑  ⛔  🚫  ⚠ ✸ ☞** to pull attention here.

(I'm serious: lots of times comments are just _ignored_ because the programmer's eye has been trained not to see them. _This_ comment is too important for that.)

Reply:
Done!